src/pages/documentation/getting-started/configure-datastreams.md

```diff
@@ -0,0 +1,40 @@
+# Configure datastreams
+
+After you've created a schema and its associated dataset, you may now configure datastreams (also referred to as Edge configurations or Edge Network configurations).
+
+<InlineAlert variant="warning" slots="text"/>
+
+To create a datastream, your organization must be provisioned for this feature in the Data Collection UI. Please contact your Adobe Customer Success Manager (CSM) to be added to the allow list.
+
+<InlineAlert variant="info" slots="text"/>
+
+The Datastreams tool is available to customers on the _allow list_ regardless whether they use the Data Collection UI for web tag management or to manage mobile app configuration. <br/><br/>Users will require **Develop** permissions for the Data Collection UI. See the [user permissions](https://experienceleague.adobe.com/docs/experience-platform/tags/admin/user-permissions.html) article for more details.
```

Comment:
I think we should use present tense here: Users will require ... -> Users require ...